test/e2e/v3/plugin_cluster_test.go

```diff
@@ -166,7 +166,7 @@ func Run(kbc *utils.TestContext) {
 	_, err = kbc.Kubectl.Command(
 		"create", "clusterrolebinding", fmt.Sprintf("metrics-%s", kbc.TestSuffix),
 		fmt.Sprintf("--clusterrole=e2e-%s-metrics-reader", kbc.TestSuffix),
-		fmt.Sprintf("--serviceaccount=%s:default", kbc.Kubectl.Namespace))
+		fmt.Sprintf("--serviceaccount=%s:e2e-%s-controller-manager", kbc.Kubectl.Namespace, kbc.TestSuffix))
```

Comment:
same here. It shows not required. I'd prefer as it is in go/v2, ansible/helm. 
It has no reason to be changed.